lib/python/rose/macros/__init__.py

```diff
@@ -42,8 +42,7 @@ class DefaultTransforms(rose.macro.MacroTransformerCollection):
     def __init__(self):
```

Comment:
Not related to the current PR changes but the 'inspect' module seems to be an unused import for this file unless it is utilised in a non-obvious way (not sure how to comment against the relevant line as I cannot seem to comment the usual way on a line in expanded view.)